core/src/main/java/de/valtech/aecu/core/groovy/console/bindings/impl/ContentUpgradeImpl.java

```diff
@@ -66,19 +67,25 @@ public ContentUpgrade forChildResourcesOf(@Nonnull String path) {
     }
 
     @Override
-    public ContentUpgrade forDescendantResourcesOf(@Nonnull String path) {
+    public ContentUpgrade forDescendantResourcesOf(@Nonnull String path, boolean includeRootResource) {
```

Comment:
How about another forDescendantResourcesOf(String) that sets includeRoot to true?

Reply:
Including the root node might be confusing since IsDescendantOf in SQL2 does not include the root node of the query path. Maybe o different name would be more suitable, like forSubtreeAt?

Reply:
Yes, how about "forResourcesInSubtree"?

Reply:
perfect :)